test/dx-clone.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+const config = require('config');
+const client = require('./lib/client').client;
+const nav = require('./lib/nav/navigation')(client, config);
+const micis = require('./lib/auth/micis')(client);
+const should = require('should');
+
+describe('Test data exchange functionality for cloning a request', function() {
+  this.timeout(config.defaultTimeout);
+
+  before('initialize', function(done) {
+    client.clientReady.then(function boot() {
+      if (!micis.loggedOn) {
+        micis.logon();
+      }
+
+      client.call(done);
+    });
+  });
+
+  it('should open Data Exchange', function(done) {
+    nav
+      .micisMenu
+      .clickNested('Browse Available Data')
+      .call(done);
+  });
+
+  it('should verify that [Previous Request Templates] have loaded', function(done) {
+    const previousRequestTemplates = '#requestMenu > option';
```

Comment:
Second example is more readable because the selector is _inside_ the client method. I’d only assign selectors to variables if they’re used repeatedly.